src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GhDampingFunction.hpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
```

Comment:
Can you rename this file to just `DampingFunction.hpp` (removing `Gh` prefix)? Then the base class to `DampingFunction`, and then I think you can also get rid of the `/// \file` comment on line 4 and 5 (we only use those if what's in the file doesn't match the file name).